### PR TITLE
feat: display state channel transactions count

### LINKED ISSUE #416: Show pagination info in state channel transactions
### Please describe the problem that your request should resolve.
Display pagination info in state channel transactions panel. Here is how you can calculate them: https://github.com/aeternity/ae_mdw/issues/1302

### PR DESCRIPTION
**Pull request by lukeromanowicz**
## Description
Resolve #416 

## Demo
image

## Checklist:

- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)  
- [x] My change does not require a change to the documentation.

### CODE REVIEW

File: src/components/StateChannelTransactionsPanel.vue [hunk 1]
```diff
@@ -31,7 +32,7 @@ import { isDesktop } from '@/utils/screen'
 import PaginatedContent from '@/components/PaginatedContent'
 
 const stateChannelDetailsStore = useStateChannelDetailsStore()
-const { stateChannelTransactions } = storeToRefs(stateChannelDetailsStore)
+const { stateChannelTransactions, stateChannelTxCount } = storeToRefs(stateChannelDetailsStore)
```
Comment (janmichek) [inline src/components/StateChannelTransactionsPanel.vue:35]: I do not really favor mixing `tx` and `transactions`. The logical question is whether it the same or a different kind of transaction. If it's the same, why is it called differently?

**Review by janmichek — CHANGES_REQUESTED** [review]
Works nicely, just giving one code suggestion

File: src/components/StateChannelTransactionsPanel.vue (hunk 1 above)
Comment (lukeromanowicz) [reply src/components/StateChannelTransactionsPanel.vue]: That is a fair point, I copied Tx from some other function name but it looks like Transactions is the right chunk

**Review by CharisSiarampalis — APPROVED** [review]

**Review by janmichek — APPROVED** [review]
Perfecto 👌

**Pull request merged**